packages/wrangler/src/deploy/deploy.ts

```diff
@@ -953,6 +954,10 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 		}
 	}
 
+	if (config.containers !== undefined && props.dryRun) {
+		await buildContainers(config, workerTag ?? "worker-tag");
+	}
+
 	if (props.dryRun) {
```

Comment:
if `config.containers !== undefined && props.dryRun` is truthy, both this `if` block and the one above will be executed. Is that what we want?

Reply:
If you refer to the `deployContainers` call, the dry run `if` statement makes the function do an early return, right?

Reply:
I agree that this change makes a lot of sense!